### PR TITLE
docs: remove line-wrapping

### PR DESCRIPTION
**Pull request by crawford**
Per CoreOS documentation standards, use one line per block of text rather than
wrapping at 80 columns.

/cc @joshix

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,7 +56,4 @@ The format can be described more formally as follows:
 <footer>
 ```
 
-The first line is the subject and should be no longer than 70 characters, the
-second line is always blank, and other lines should be wrapped at 80 characters.
-This allows the message to be easier to read on GitHub as well as in various
-git tools.
+The first line is the subject and should be no longer than 70 characters, the second line is always blank, and other lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
```
Comment (crawford) [inline CONTRIBUTING.md]: This is only slightly hypocritical. ;)

File: doc/configuration.md [hunk 2]
```diff
@@ -1,108 +1,62 @@
...
-                           typically referenced by the /dev/disk/by-* symlinks.
-    - **_wipeTable_** (boolean): whether or not the partition tables shall be
-                                 wiped. When true, the partition tables are
-                                 erased before any further manipulation.
-                                 Otherwise, the existing entries are left
-                                 intact.
-    - **_partitions_** (list of objects): the list of partitions and their
-                                          configuration for this particular
-                                          disk.
+- **_storage_** (object): describes the desired state of the system's storage devices.
```
Comment (joshix) [inline doc/configuration.md]: `s/^- /* /` throughout these bullet lists. Most existing documents use the `*` marker for lists and we are going to enshrine that in the style guide.

File: doc/getting-started.md [hunk 3]
```diff
@@ -1,38 +1,23 @@
 # Getting Started with Ignition #
 
-Ignition uses a JSON config to represent the set of changes to be made. The
-format of this config is detailed [in the spec][config spec]. One of the most
-important parts of this config is the version number. This **must** match the
-version number accepted by Ignition. If the config version isn't accepted by
-Ignition, Ignition will fail to run and prevent the machine from booting. This
-can be seen by inspecting the console output of the failed instance. For more
-information, check out the [troubleshooting section](#troubleshooting).
+Ignition uses a JSON config to represent the set of changes to be made. The format of this config is detailed [in the spec][config spec]. One of the most important parts of this config is the version number. This **must** match the version number accepted by Ignition. If the config version isn't accepted by Ignition, Ignition will fail to run and prevent the machine from booting. This can be seen by inspecting the console output of the failed instance. For more information, check out the [troubleshooting section](#troubleshooting).
```
Comment (joshix) [inline doc/getting-started.md]: `s/JSON config/JSON configuration file/` might clarify this, at least as you introduce the term.

File: doc/getting-started.md [hunk 4]
```diff
@@ -1,38 +1,23 @@
...
 
 [config spec]: configuration.md
 
 ## Providing a Config ##
 
-Ignition will look in different places for its configuration and metadata,
-depending on the platform on which it is running. A full list of the
-[supported platforms] and their data sources is provided for reference.
+Ignition will look in different places for its configuration and metadata, depending on the platform on which it is running. A full list of the [supported platforms] and their data sources is provided for reference.
```
Comment (joshix) [inline doc/getting-started.md]: Ignition will choose where to look for configuration based on the underlying platform. A list of [Ignition's supported platforms](supported-platforms.md) and metadata sources is provided for reference.

File: doc/configuration.md (hunk 2 above)
Comment (crawford) [inline doc/configuration.md]: Roger

File: doc/getting-started.md [hunk 5]
```diff
@@ -1,38 +1,23 @@
...
 
-The configuration will need to be passed to Ignition via the designated data
-source.
+The configuration will need to be passed to Ignition via the designated data source.
```
Comment (joshix) [inline doc/getting-started.md]: The configuration must be passed to Ignition through the designated data source.

File: doc/supported-platforms.md [hunk 6]
```diff
@@ -2,21 +2,9 @@
...
-  `coreos.config.url` kernel parameter to provide a URL to the configuration.
-  The URL can use the `http://` scheme to specify a remote config or the
-  `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
-- [Amazon EC2](https://coreos.com/os/docs/latest/booting-on-ec2.html) -
-  Ignition will read its configuration from the userdata and append the SSH
-  keys listed in the instance metadata.
-- [Microsoft Azure](https://coreos.com/os/docs/latest/booting-on-azure.html) -
-  Ignition will read its configuration from the custom data provided to the
-  instance. SSH keys are handled by the Azure Linux Agent.
+- [Bare Metal](https://coreos.com/os/docs/latest/installing-to-disk.html) - Use the `coreos.config.url` kernel parameter to provide a URL to the configuration. The URL can use the `http://` scheme to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
```
Comment (joshix) [inline doc/supported-platforms.md]: `s/^- /* /g`

**Comment by joshix** [comment]
LGTM, and thanks for all the `sed`.

**Pull request merged**